public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/grafanaApp.ts

```diff
@@ -8,9 +9,13 @@ import { isOnCallReceiver } from './onCall/onCall';
 import { AmRouteReceiver, GrafanaAppReceiverEnum, GRAFANA_APP_PLUGIN_IDS, ReceiverWithTypes } from './types';
 
 export const useGetAppIsInstalledAndEnabled = (grafanaAppType: GrafanaAppReceiverEnum) => {
-  // fetches the plugin settings for this Grafana instance
-  const plugin: CatalogPlugin | undefined = useGetSingleLocalWithoutDetails(GRAFANA_APP_PLUGIN_IDS[grafanaAppType]);
-  return plugin?.isInstalled && !plugin?.isDisabled && plugin?.type === 'app';
+  const {
+    loading,
+    error,
+    value: plugin,
+  } = useAsync(() => getPluginSettings(GRAFANA_APP_PLUGIN_IDS[grafanaAppType], { showErrorAlert: false }));
```

Comment:
I think you should be able to use the [usePluginBridge](https://github.com/grafana/grafana/blob/909ec67c56449ebbb07a4864e74778d8a2044a08/public/app/features/alerting/unified/components/PluginBridge.tsx#L44) hook without having to call `getPluginSettings` manually.